saffron/e2e-test.sh

```diff
@@ -11,8 +11,34 @@ SRS_ARG=""
 if [ $# -eq 2 ]; then
    SRS_ARG="--srs-filepath $2"
 fi
+
 ENCODED_FILE="${INPUT_FILE%.*}.bin"
 DECODED_FILE="${INPUT_FILE%.*}-decoded${INPUT_FILE##*.}"
+PERTURBED_FILE="${INPUT_FILE%.*}_perturbed${INPUT_FILE##*.}"
+ENCODED_DIFF_FILE="${ENCODED_FILE%.*}_diff.bin"
+DECODED_PERTURBED_FILE="${PERTURBED_FILE}-decoded${INPUT_FILE##*.}"
+
+compare_files() {
+   local file1="$1"
+   local file2="$2"
+   
+   echo "Comparing files..."
+   if cmp -s "$file1" "$file2"; then
+       echo "✓ Success: Files are identical"
+   else
+       echo "✗ Error: Files differ"
+       echo "Keeping files for inspection"
+       diff "$file1" "$file2"
+       exit 1
+   fi
+}
+
+perturb_bytes() {
```

Comment:
Are you sure this works?

Reply:
wow, no it wasn't. I have replaced it with a python script instead because perl is like hieroglyphics to me. But actually this makes me realize that the user will should keep all the commitments on their end, not just the folded commitment with powers of `alpha`. As of right now, they cannot update them without recomputing which is dumb.